index.html

```diff
@@ -18,25 +18,57 @@ <h1 class="display-4 text-center">
         <i class="fas fa-book-open text-primary"></i> My<span class="text-primary">Book</span>List
       </h1>
       <form id="book-form">
-        <div class="form-group">
+        <div class="form-group mb-2">
           <label for="title">Title</label>
           <input type="text" id="title" class="form-control">
+          <span class="text small text-danger titleError">Title is required</span>
         </div>
-        <div class="form-group">
+        <div class="form-group mb-2">
           <label for="author">Author</label>
           <input type="text" id="author" class="form-control">
+          <span class="text small text-danger authorError">Author is required</span>
         </div>
-        <div class="form-group">
+
+        <div class="form-group mb-2">
+          <label for="authorEmail">Author Email</label>
+          <input type="text" id="authorEmail" class="form-control">
+          <span class="text small text-danger emailError">Email is required</span>
+        </div>
+
+        <div class="form-group mb-2">
+          <label for="author-phone">Auther Phone</label>
+          <input type="text" id="author-phone" class="form-control">
```

Comment:
type="tel" and pattern="[0-9]{3}-[0-9]{2}-[0-9]{3}" to be added so that this field is being considered as telephone and proper pattern is being applied.